pkg/ee/validation/machine/providers.go

```diff
@@ -46,8 +46,30 @@ import (
 	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func GetMachineResourceUsage(ctx context.Context, userClient ctrlruntimeclient.Client, machine *clusterv1alpha1.Machine,
-	caBundle *certificates.CABundle) (*ResourceDetails, error) {
+const (
+	// GCP credential env.
+	envServiceAccount = "GOOGLE_SERVICE_ACCOUNT"
+	// Azure credential env.
+	envClientID       = "AZURE_CLIENT_ID"
+	envClientSecret   = "AZURE_CLIENT_SECRET"
+	envTenantID       = "AZURE_TENANT_ID"
+	envSubscriptionID = "AZURE_SUBSCRIPTION_ID"
+	// Openstack credential env.
+	envOSUsername                    = "OS_USER_NAME"
+	envOSPassword                    = "OS_PASSWORD"
+	envOSDomain                      = "OS_DOMAIN_NAME"
+	envOSApplicationCredentialID     = "OS_APPLICATION_CREDENTIAL_ID"
+	envOSApplicationCredentialSecret = "OS_APPLICATION_CREDENTIAL_SECRET"
+	envOSToken                       = "OS_TOKEN"
+	envOSProjectName                 = "OS_PROJECT_NAME"
+	envOSProjectID                   = "OS_PROJECT_ID"
```

Comment:
NIT: maybe add `envGoogleServiceAccount` , `envAzureClientID` and such so its clear

Reply:
Yeah, I think adding more context may ease code analysis.